sh.cider.Cider.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <content_rating type="oars-1.0" />
+  <id>sh.cider.Cider</id>
+  
+  <name>Cider</name>
+  <summary>An electron based Apple Music front-end</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-1.0-or-later</project_license>
+
+  <url type="homepage">https://cider.sh/</url>
+  <url type="bugtracker">https://github.com/ciderapp/Cider/issues</url>
+  <url type="donation">https://ko-fi.com/cryptofyre</url>
+  
+  <description>
+    <p>
+      Project Cider. A new look into listening and enjoying Apple Music in style and performance.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">cider.desktop</launchable>
```

Comment:
The desktop file name has to follow app ID, including this field.

Reply:
but isn't this handled by the `rename-desktop-file`? Doing so make it easy to have this file managed upstream as is, until the .desktop file names also match the XDG spec.

Reply:
I'd be surprised if it is.

Reply:
given that when it is wrong you get the mysterious message about not being to find the icons after appstream-compose ran.

Reply:
So what do we do about this do I change it?